csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -111,6 +97,25 @@ public DependencyManager(string srcDir, IDependencyOptions options, ILogger logg
                 DownloadMissingPackages(allNonBinaryFiles);
             }
 
+            var existsNetCoreRefNugetPackage = false;
+            var existsNetFrameworkRefNugetPackage = false;
+
+            // Find DLLs in the .Net / Asp.Net Framework
+            if (options.ScanNetFrameworkDlls)
```

Comment:
Maybe add a comment that this section needs to be after the `UseNuGet` section as we now attempt to use framework dll's that might be downloaded as a part of the solution/project restore.